mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.tsx

```diff
@@ -270,6 +297,96 @@ class ShowArtifactLoggedModelView extends Component<Props, State> {
     );
   }
 
+  renderServingPayload(servingInput?: string) {
+    if (servingInput === null) {
+      return (
+        <div className="code">
```

Comment:
Can we verify if this is the same rendering component that we get when data cannot be fetched from other locations within the MLflow UI?

Reply:
that makes sense, though i think it may be better as a follow-up PR since i think a few things would have to change:
1. separating the left and right sections to let them scroll independently
2. limiting the height of the code snippets
3. probably replacing these code snippets with the `<CodeSnippet>` component